test/markup/erlang/char_literal.expect.txt

```diff
@@ -0,0 +1,16 @@
+Character = <span class="hljs-string">$a</span>
+TabCharacter = <span class="hljs-string">$\t</span>
+TabCharacterDecimal = <span class="hljs-string">$\011</span>
+IncorrectMixed = <span class="hljs-string">$a</span>85
+IncorrectTooLongChar = <span class="hljs-string">$a</span>a
+IncorrectTooLongNumber = <span class="hljs-string">$1</span>23
+IncorrectTooLongEscapeNumber = <span class="hljs-string">$\123</span>4
```

Comment:
Probably should try not to highlight these at all if they are invalid... could a look-ahead be used to look for following whitespace, etc?

Reply:
The reason I'm doing this is because it's a syntax error in the language to write that anyway.
I don't know what your policy is on highlighting syntax errors, but technically the error doesn't occur until *after* the character literal.

If you insist, I will add a look-ahead.

Reply:
> I don't know what your policy is on highlighting syntax errors

Generally we don't consider them at all... thanks for the reminder.  That said I don't think we'd need or want these 4 tests cases then - even if we left them with an explanatory comment I'm not sure what purpose they would service - since they are literally syntax error examples.

So I think just remove those 4 examples and we're good.